### ISSUE TITLE
Broken code in production mode

### ISSUE THREAD

**Issue by scf2k** [bem/bem-xjst#62]
```
__$ctx.i-global = __$ctx.extend(_$jps__$1672, _$jenv__$1680);
```

Does not work because of minus in `i-global`

**Comment by vkz** [comment]
do u know where this i-global is coming from? Can I see the templates u're compiling, pls. Need more context.

**Comment by vkz** [comment]
@indutny fyi

it seems that compiler handles context arg to `applyNext` incorrectly, i.e. doesn't check if the prop name can syntactically be used as direct look up on the object:

```js
applyNext({ 'i-global': this.extend(ps, env) });
```

compiles into:
```js
        var __$l6__$23 = __$ctx.i-global;
        __$ctx.i-global = __$ctx.extend(_$0ps__$7, _$0env__$15);
        applyc(__$ctx, __$ref);
```

**Comment by indutny** [comment]
Should be fixed with xjst@1.5.1 . Thank you for reporting it!
